packages/react-drag-drop/src/components/DragDrop/DragButton.tsx

```diff
@@ -1,27 +1,39 @@
 import * as React from 'react';
 import { css } from '@patternfly/react-styles';
+import { Button } from '@patternfly/react-core';
 import dragButtonStyles from '@patternfly/react-styles/css/components/DataList/data-list';
-import buttonStyles from '@patternfly/react-styles/css/components/Button/button';
 import GripVerticalIcon from '@patternfly/react-icons/dist/esm/icons/grip-vertical-icon';
 
-export interface DragButtonProps extends React.HTMLProps<HTMLButtonElement> {
+export interface DragButtonProps extends Omit<React.HTMLProps<HTMLButtonElement>, 'size'> {
   /** Additional classes added to the drag button */
   className?: string;
   /** Sets button type */
   type?: 'button' | 'submit' | 'reset';
```

Comment:
Was there a reason this prop is included on the DragButton? I'd imagine we would never want the DragButton to act as a "submit" or "reset" button. cc @kmcfaul